### DISCUSSION TITLE
How can you get the next badge?

### DISCUSSION THREAD

**Discussion by EdisonGualle** [community/community#78278] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can you get the next badge?

![image](https://github.com/community/community/assets/145297761/2ffc2551-7b7d-41ee-b232-771cc05fcefc)

**Comment by queenofcorgis** [comment]
Hi @EdisonGualle ,

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
